c++/src/kj/string-test.c++

```diff
@@ -24,6 +24,7 @@
 #include <string>
 #include "vector.h"
 #include <locale.h>
+#include <cstdint>
```

Comment:
In KJ code we prefer not to use the C++ wrappers for C headers, we just use the C headers directly.

```suggestion
#include <stdint.h>
```

(Technically the C++ wrapper header is supposed to define all symbols within the `std` namespace. If it actually did so, then this fix presumably wouldn't work, because we don't prefix `uint8_t` with `std::`. Of course, in practice this requirement that symbols be defined in `std::` doesn't seem to be followed -- libstdc++ goes ahead and puts everything in the global scope anyway. But in that case, what's the point of using these wrapper headers? We might as well use the original C headers. If we use the wrappers then we risk that someday we want to port the code to a platform that actually does the namespacing correctly... and it doesn't work!)